python/pyspark/ml/feature.py

```diff
@@ -2385,7 +2385,31 @@ class RFormula(JavaEstimator, HasFeaturesCol, HasLabelCol):
     |0.0|0.0|  a|[0.0,1.0]|  0.0|
     +---+---+---+---------+-----+
     ...
-    >>> rf.fit(df, {rf.formula: "y ~ . - s"}).transform(df).show()
+    >>> model = rf.fit(df, {rf.formula: "y ~ . - s"})
```

Comment:
Oh, sorry, I originally thought this was a new fit() being added, and I wanted to avoid unnecessary model fitting.  I now see it was already there, so it's fine either way.